conf/.php-cs-fixer.php

```diff
@@ -60,6 +60,23 @@
         'no_blank_lines_after_class_opening' => true,
         'no_php4_constructor' => true,
         'ordered_class_elements' => [
+            'order' => [
+                'use_trait',
+                'constant_public',
+                'constant_protected',
+                'constant_private',
+                'property_public',
+                'property_protected',
+                'property_private',
+                'construct',
+                'destruct',
+                'magic',
+                'phpunit',
+                'method_abstract',
```

Comment:
I'm thinking higher still.

I consider abstract method declarations on a par with traits tbh.  I'm honestly not sure which I'd put first.

Given you literally can't use a class without knowing about them.

Reply:
Hm, makes sense, ok.